### PR TITLE
add Ddoc comments to root files

### PR DESCRIPTION
**Pull request by WalterBright**
not very interesting

### CODE REVIEW

**Comment by 9rnsr** [comment]
Auto-merge toggled on

File: src/root/filename.d [hunk 1]
```diff
@@ -69,7 +72,11 @@ struct FileName
     }
 
     /************************************
-     * Return !=0 if absolute path name.
+     * Determine if path is absolute.
+     * Params:
+     *  name = path
+     * Return:
```
Comment (Geod24) [inline src/root/filename.d]: `Returns`

File: src/root/filename.d [hunk 2]
```diff
@@ -689,6 +704,6 @@ struct FileName
 
     extern (C++) const(char)* toChars() const
     {
-        return cast(char*)str; // toChars() should really be const
+        return str;
```
Comment (Geod24) [inline src/root/filename.d]: I know this is trivial, but please don't mix code changes into commits marked as documentation only changes.

File: src/root/response.d [hunk 3]
```diff
@@ -1,21 +1,14 @@
 /**
- * Compiler implementation of the
- * $(LINK2 http://www.dlang.org, D programming language).
+ * Compiler implementation of the D programming language
+ * http://dlang.org
```
Comment (Geod24) [inline src/root/response.d]: Why ?

File: src/root/response.d [hunk 4]
```diff
@@ -1,21 +1,14 @@
...
- * Copyright:
- * Some portions copyright (c) 1994-1995 by Symantec
- * Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
- *
- * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
- *
- * License:
- * This source file is made available for personal use
- * only. The license is in backendlicense.txt
- * For any other uses, please contact Digital Mars.
```
Comment (Geod24) [inline src/root/response.d]: Also, why does it replace meaningful documentation with copy-pasta ?

**Comment by Geod24** [comment]
@braddr : Any chance you could fetch the others' test status from the auto-tester and wait on those ?
E.g. this was merged while the Travis CI build is still in progress, and it actually failed (due to network issue, but still): https://travis-ci.org/D-Programming-Language/dmd/builds/119478997

**Comment by Passw** [comment]
@WalterBright How about updating all ```*.d``` files to use Ddoc style comments ?

**Pull request merged**